plugins/modules/docker_image_build.py

```diff
@@ -110,6 +114,38 @@
       - never
       - always
     default: never
+  secret:
+    description: Secrets to expose to the build.
+    type: list
+    elements: dict
+    suboptions:
+      id:
+        description: The secret identifier.
+        type: str
+        required: True
+      src:
+        description: Source path of the secret.
+        type: str
+      value:
+        description: Value of the secret.
+        type: str
+      type:
+        description: Type of the secret.
+        type: str
+        choices: ['file', 'password']
+        required: True
+  load:
+    description:
+      - Load the built image into Docker's local image store.
+      - Cannot be used together with C(push).
+    type: bool
+    default: true
```

Comment:
Why is this default `true`? Isn't this a breaking change?

Reply:
I usually prefer by default store it local but will remove default so who want can specifiy

Reply:
changed it to false